schedule/containers/sle_image_on_centos.yaml

```diff
@@ -0,0 +1,9 @@
+name:           sle_image_on_centos
+description:    |
+    Run sle container on Centos
```

Comment:
Could you please add here
```
Maintainer: qa-c@suse.de.
```

Reply:
After our discussion i am leaving it as it is. Maintainer is showed in each module